onnxscript/rewriter/generic_pattern.py

```diff
@@ -328,6 +332,164 @@ def _match_backward(
             print(f"[GenericPatternMatcher._match_backward] add {match_count} nodes")
         return match_count
 
+    def _match_values_forward(
```

Comment:
Did you change anything here? It is difficult to check since the code was moved.